echopype/tests/calibrate/test_calibrate_ek80.py

```diff
@@ -306,10 +306,7 @@ def test_ek80_BB_power_compute_Sv(
         encode_mode=encode_mode,
     )
     pyel_vals = pyel_BB_p_data["sv_data"]
-    if dask_array:
-        ep_vals = ds_Sv["Sv"].sel(channel=ch_sel).squeeze().data.compute()
-    else:
-        ep_vals = ds_Sv["Sv"].sel(channel=ch_sel).squeeze().data
+    ep_vals = ds_Sv["Sv"].sel(channel=ch_sel).squeeze().data.compute()
```

Comment:
Does this mean that right now with the `dask_array=False` test case the data is still lazy-loaded? Asking because I seem to remember the original intention of this is to see if lazy-loading works explicitly, but I could also just be misremembering things.

Reply:
Ah yeah, it's because of the `.chunk({"range_sample": -1, "channel": -1})` that's now in the convolution. Let me make this chunk conditional on whether or not the backscatter variable is already chunked.